src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/OnepageSuccessAction.php

```diff
@@ -28,9 +28,8 @@ class Shopgate_Cloudapi_Model_Frontend_Observer_OnepageSuccessAction
      */
```

Comment:
can you add ```@throws Mage_Core_Exception``` plz :)

Reply:
sure ;)

Reply:
Oops, I meant in the SalesOrderAfter.php execute, it's not thrown in this observer ^_^ my bad!